<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JPEGMainWindow</class>
 <widget class="QMainWindow" name="JPEGMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPEG测试程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> JPEG压缩测试</string>
     </attribute>
     <widget class="QLineEdit" name="originalPicturePath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>251</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>选择计算方式</string>
      </property>
      <widget class="QRadioButton" name="fpgaRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>FPGA</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="armRadioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ARM</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>信息</string>
      </property>
      <widget class="QLabel" name="compressedLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>压缩后大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="methodLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>实际计算方式:</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>计算时间:</string>
       </property>
      </widget>
      <widget class="QLabel" name="originalLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>原图大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="originalLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="methodLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>82</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="timeLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>82</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="compressedLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>82</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="selectPathButton">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>20</y>
        <width>121</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>选择BMP图片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>加解密后路径:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pictureCompressionButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>300</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>开始图像压缩</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="originalPicture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>211</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="compressedPicture">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>201</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>82</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>原图片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>压缩后图片</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>461</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>JPEG头信息</string>
      </property>
      <widget class="QLabel" name="originalWidth">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>水平方向像素   :</string>
       </property>
      </widget>
      <widget class="QLabel" name="originalHeight">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>60</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>垂直方向像素   :</string>
       </property>
      </widget>
      <widget class="QLabel" name="compressionWidth">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>90</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>缩略图水平像素:</string>
       </property>
      </widget>
      <widget class="QLabel" name="compressionHeight">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>缩略图垂直像素:</string>
       </property>
      </widget>
      <widget class="QLabel" name="originalWidthNum">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="originalHeightNum">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="compressionWidthNum">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>90</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="compressionHeightNum">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>120</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTextEdit" name="infomationText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
